File: C/lt_01_estseq/lt01_estseq_13.c
// Nome do Programador:
// Descrição:
#include <stdio.h>

int main() {

  float kilos;

  do {

    printf("Insira o peso em Kg: ");
    scanf("%f", &kilos);
    if (kilos < 0) {

      printf("Peso inválido!\n");
    }

  } while (kilos < 0);

  kilos = (kilos * 1000) / 50;

  printf("A comida durara por %.2f dias.\n", kilos);

  return 0;
}

File: Java/LT01_ESTSEQ/LT01_ESTSEQ_13.java
/*
 * Nome Do Programador: Lucas Sartorelli
 * Data: 13/02/2025
 * Descrição: Receba a quantidade de alimento em quilos. 
 * Calcule e mostre quantos dias durará esse alimento sabendo que a pessoa consome 50g ao dia.
 */
import javax.swing.JOptionPane;
public class LT01_ESTSEQ_13 {
	public static void main (String args[]) {
		Double kilos;
		kilos = Double.parseDouble(JOptionPane.showInputDialog("Insira a quantidade de Comida em Kg: "));
		kilos /= 0.05;
		JOptionPane.showMessageDialog(null, "Este alimento durará " +kilos +" dias.");
	}
}
